lib/takeaway.rb

```diff
@@ -0,0 +1,24 @@
+class Takeaway
+  attr_reader :menu, :price, :number, :total_sum
+
+  def initialize
+    @menu = ["Pasta: £3.50", "lasagna: £4.50", "water: £1.50"]
+    # @price = price
```

Comment:
I like how you're thinking ahead to what you may want in the future. It shows that you have thought about the user stories and you're prepared to adapt your code in the future.